pkg/resources/resources.go

```diff
@@ -0,0 +1,52 @@
+package resources
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// DeletePods deletes all the pods from the unhealthy node
```

Comment:
Nit: in that context this method just deletes all the pods from the node (i.e we don't know whether the node is unhealthy or not)